flexget/plugins/sites/newznab.py

```diff
@@ -1,125 +1,608 @@
+    # NOTE: will most likely break well behaving indexers, so use in a separate task/entry or use nzbhydra
+    newznab:
+        api_server_url: https://api.nzbindexer.com
+        category: tv
+        force_quotes: yes
     """
 
     schema = {
         'type': 'object',
         'properties': {
-            'category': {'type': 'string', 'enum': ['movie', 'tvsearch', 'tv', 'music', 'book']},
-            'url': {'type': 'string', 'format': 'url'},
-            'website': {'type': 'string', 'format': 'url'},
-            'apikey': {'type': 'string'}
+            'api_server_url': {'type': 'string', 'format': 'url'},
+            'api_key': {'type': 'string'},
+            'category': one_or_more({
+                'oneOf': [
+                    {'type': 'integer'},
+                    {'type': 'string', 'enum': list(CATEGORIES)},
+                ]}, unique_items=True),
+            'custom_query': {'type': 'string'},
+            'force_quotes': {'type': 'boolean', 'default': False},
+            'maxage': {
+                'oneOf': [
+                    {'type': 'string', 'format': 'interval'},
+                    {'type': 'string', 'enum': ['auto']}
+                ]
+            },
+            'use_metadata': {
+                'oneOf': [
+                    {'type': 'boolean'},
+                    one_or_more({'type': 'string', 'enum': list(PLUGIN_LOOKUP_MAP)}, unique_items=True)
+                ]
+            },
         },
-        'required': ['category'],
+        'required': ['api_server_url', 'category'],
         'additionalProperties': False
     }
 
-    def build_config(self, config):
-        log.debug(type(config))
+    def prepare_config(self, config):
+        if 'use_metadata' in config:
+            if 'plugins_list' not in config:
+                if isinstance(config['use_metadata'], str):
+                    config['use_metadata'] = [config['use_metadata']]
+                if isinstance(config['use_metadata'], list):
+                    plugins = []
+                    for name in config['use_metadata']:
+                        plugins.append(PLUGIN_LOOKUP_MAP[name])
+                        config['plugins_list'] = plugins
+        if 'category' in config:
+            if 'category_string' not in config:
+                if not isinstance(config['category'], list):
+                    config['category'] = [config['category']]
+                categories = config['category']
+                # Convert named categories to its respective categories id number
+                categories = [c if isinstance(c, int) else CATEGORIES[c] for c in categories]
+                if len(categories) > 0:
+                    config['category_string'] = ','.join(str(c) for c in categories)
+        return config
+
+    def _is_tv_entry(self, entry):
+        if any(entry.get(field) for field in ['series_name', 'tvdb_id', 'tvrage_id', 'tvmaze_series_id', 'trakt_id']):
```

Comment:
There is a way to do this without hard coding this list here.

````
[p.instance.movie_identifier for p in plugin.get_plugins(group='series_metainfo')]
````
(you can search this for usages)

Might not include series_name though, but at least gets mostly everything dynamically. You need to apply this other places too, we don't want to manually maintain these lists across numerous plugins.